src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -36,8 +37,13 @@
 public class SubscriptionService implements ISubscriptionService {
 
     @Value("${spring.application.name}") private String SpringApplicationName;
+    
+    @Value("${spring.data.mongodb.database}") private String dataBaseName;
+    
+    @Value("{subscription.collection.repeatFlagHandlerName}") private String RepeatFlagHandlerCollection;
```

Comment:
variables start with lower case. Change SpringApplicationName too since you modify the file anyway